src/Summoner/Template.hs

```diff
@@ -298,24 +298,33 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
 
     -- create README template
     readme :: Text
-    readme =
+    readme = 
         [text|
         # $repo
 
-        [![Hackage]($hackageShield)]($hackageLink)
-        [![$license license](${licenseShield})](${licenseLink})
-        $stackBadges
-        $travisBadge
-        $appVeyorBadge
-
+        $hackage
+        $licenseBadge
+        $maybeSomeBadges
```

Comment:
The idea is to not use `neat-interpolation` at all for `readme`. This file is somewhat special. And since we have a lot of conditionals here, it's better to use combination of `intercalate` function with list comprehension to not have empty lines at all but still have nice structure for the `README.md` file